src/main/java/uk/gov/hmcts/ccd/domain/service/callbacks/CallbackService.java

```diff
@@ -119,6 +119,7 @@ private Integer secondsToMilliseconds(final Integer seconds) {
             requestFactory.setConnectionRequestTimeout(secondsToMilliseconds(timeout));
             requestFactory.setReadTimeout(secondsToMilliseconds(timeout));
             requestFactory.setConnectTimeout(secondsToMilliseconds(timeout));
+            restTemplate.setRequestFactory(requestFactory);
             return Optional.ofNullable(
```

Comment:
I am now confused.  A while back, restTemplate was created inside this function.  Then it was injected.  Now we are changing the properties of this restTemplate.  Should we create a restTemplate as per call back??  Just a thought

Reply:
Your point may be right, but I think we should discuss it out of the scope of this bug fix. Perhaps with another task to validate the need for that.
Just for the sake of completing this task.